content/tutorials/manage-the-rippled-server/configuration/run-rippled-as-a-validator.md

```diff
@@ -1,74 +1,191 @@
+
+For more information about using the `validator-keys` tool, see the [Validator Keys Tool Guide](https://github.com/ripple/validator-keys-tool/blob/master/doc/validator-keys-tool-guide.md).
+
+1. Generate a validator key pair.
 
         $ /opt/ripple/bin/validator-keys create_keys
 
-    **Warning:** Store the generated `validator-keys.json` key file in a secure but recoverable location, such as an encrypted USB flash drive. Do not modify its contents.
+    **Warning:** Store the generated `validator-keys.json` key file in a secure but recoverable location, such as an encrypted USB flash drive. Do not modify its contents. ***TODO: Is the main point here that this is an offline location? Specifically, is it not secure enough to keep it in the default location it is generated in? Like if I'm running my server in AWS EC2 - should I move the json file to my flash drive and delete it from EC2?***
 
-3. Generate a validator token and edit your `rippled.cfg` file to add the `[validator_token]` value.
+2. Generate a validator token.
 
         $ /opt/ripple/bin/validator-keys create_token --keyfile /path/to/your/validator-keys.json
 
-    If you had previously configured your validator without using the `validator-keys` tool, you must also delete the `[validation_seed]` from your `rippled.cfg` file. This changes your validator public key.
+      Sample output:
+
+        Update rippled.cfg file with these values:
+
+        # validator public key: nHUtNnLVx7odrz5dnfb2xpIgbEeJPbzJWfdicSkGyVw1eE5GpjQr
+
+        [validator_token]
+        eyJ2YWxpZGF0aW9uX3NlY3J|dF9rZXkiOiI5ZWQ0NWY4NjYyNDFjYzE4YTI3NDdiNT
+        QzODdjMDYyNTkwNzk3MmY0ZTcxOTAyMzFmYWE5Mzc0NTdmYT|kYWY2IiwibWFuaWZl
+        c3QiOiJKQUFBQUFGeEllMUZ0d21pbXZHdEgyaUNjTUpxQzlnVkZLaWxHZncxL3ZDeE
+        hYWExwbGMyR25NaEFrRTFhZ3FYeEJ3RHdEYklENk9NU1l1TTBGREFscEFnTms4U0tG
+        bjdNTzJmZGtjd1JRSWhBT25ndTlzQUtxWFlvdUorbDJWMFcrc0FPa1ZCK1pSUzZQU2
+        hsSkFmVXNYZkFpQnNWSkdlc2FhZE9KYy9hQVpva1MxdnltR21WcmxIUEtXWDNZeXd1
+        NmluOEhBU1FLUHVnQkQ2N2tNYVJGR3ZtcEFUSGxHS0pkdkRGbFdQWXk1QXFEZWRGdj
+        VUSmEydzBpMjFlcTNNWXl3TFZKWm5GT3I3QzBrdzJBaVR6U0NqSXpkaXRROD0ifQ==
+
+3. Add `[validator_token]` and its value to your `rippled.cfg` file. ***TODO: Is it okay to also put the "# validator public key: XXXXX" text in the cfg file?***
+
+    If you previously configured your validator without the `validator-keys` tool, delete `[validation_seed]` and its value from your `rippled.cfg` file. This changes your validator public key. ***TODO: validation_seed has been disabled since 0.29.1 -- can we safely delete this text?***
+
+5. Restart `rippled`.
+
+        $ sudo systemctl restart rippled.service
+
+***TODO: how can I tell that my server is behaving as a validator? what command can tell me that my server is issuing validation messages? How can I look at a list of validations on the network and see if my validator is associated with some of them?***
```

Comment:
server_info has a pubkey_validator field, which should have your public key in it if you are validating. Your server state should also be "proposing".